package.json

```diff
@@ -26,6 +26,7 @@
     "@babel/cli": "7.16.8",
     "@babel/core": "7.16.12",
     "@babel/runtime": "7.18.6",
+    "@babel/preset-env": "7.18.9",
```

Comment:
do you know why this is needed now and why we didn't have it first?

Reply:
I succeeded  to make it work without this change. It was part of the solution I found to fix the "@babel/helper-plugin-utils broken", but apparently, it works without that change (just updating yarn.lock was enough)